### ISSUE TITLE
pip install failed

### ISSUE THREAD

**Issue by 2bithacker** [job/aggregate6#1]
```
> sudo -H pip install aggregate6
Collecting aggregate6
  Downloading aggregate6-0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/tmp/pip-build-JFgKf2/aggregate6/setup.py", line 52, in <module>
        reqs = [str(ir.req) for ir in install_reqs]
      File "/Library/Python/2.7/site-packages/pip/req/req_file.py", line 79, in parse_requirements
        "parse_requirements() missing 1 required keyword argument: "
    TypeError: parse_requirements() missing 1 required keyword argument: 'session'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/tmp/pip-build-JFgKf2/aggregate6/
```

```
> pip --version
pip 8.1.1 from /Library/Python/2.7/site-packages (python 2.7)
```

This is on Mac OS X 10.11.3.

**Comment by job** [comment]
Can you install from github and tell me if you still have the error? `sudo -H pip install git+https://github.com/job/aggregate6`

**Comment by iam-TJ** [comment]
I saw this too; it's due to pip trying to fetch the source rather than build from local. It needs:

git clone https://github.com/job/aggregate6
cd aggregate6
sudo pip install .

I also noticed some pre-requisites are not mentioned which (on Ubuntu 16.04) were required:

sudo apt install python-dev python-pip python-setuptools

**Comment by job** [comment]
Addressed in 147bfaf4bae48e3ff2788adf60215ac814a3f8e4 and 45d757de1a0d6704c292460596b9dee3d7dc1cb2
